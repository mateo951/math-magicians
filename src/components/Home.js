import React from 'react';
import style from './style/home.module.css';

const Home = () => (
  <div key="Homepage">
    <h1 className={style.title}>
      Welcome to our Page
    </h1>
    <p className={style.description}>
      Enjoy your calculations!
    </p>
  </div>
);

export default Home;
