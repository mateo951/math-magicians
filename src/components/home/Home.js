import React from 'react';
import style from './home.module.css';

const Home = () => (
  <div key="Homepage">
    <h1 className={style.title}>Welcome to our page</h1>
    <p className={style.description}>
      Enjoy your calculations!
    </p>
  </div>
);

export default Home;
