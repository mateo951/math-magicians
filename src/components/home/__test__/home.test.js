import React from "react";
import ReactDOM from 'react-dom';
import { render, screen, cleanup, fireEvent } from '@testing-library/react';
import Home from "../Home";

let div = null;
beforeEach(() => {
  div = document.createElement('div');
  document.body.appendChild(div);
  render(<Home />);
});

describe('Test rendering', () => {
  it('renders with out crashing', () => {
    ReactDOM.render(<Home />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders text correctly', () => {
    expect(screen.getByText('Welcome to our page')).toMatchSnapshot();
  });
});
