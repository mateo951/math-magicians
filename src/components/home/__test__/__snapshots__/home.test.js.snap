// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test rendering renders text correctly 1`] = `
<h1
  class="title"
>
  Welcome to our page
</h1>
`;
